addons/mail/controllers/thread.py

```diff
@@ -78,6 +78,26 @@ def read_subscription_data(self, follower_id):
     def _get_allowed_message_post_params(self):
         return {"attachment_ids", "body", "message_type", "partner_ids", "subtype_xmlid", "parent_id"}
 
+    def _prepare_post_data(self, post_data, thread, special_mentions, **kwargs):
```

Comment:
If other comments are applied, hopefully we no longer need to override or split this method.